File: testdata/override/simpleTests/foo.scala
implement foo
package test

class Foo extends b {
  <caret>
}
abstract class b {
  def foo(x: b): b
}<end>
package test

class Foo extends b {
    def foo(x: b): b = null
}
abstract class b {
    def foo(x: b): b
}

File: scala/scala-impl/testdata/rename3/settersWithoutVar2/before/tests/Foo.java
package tests;

public class Foo {
    public static void main(String[] args) {
        int b1 = Bar.b/*caret*/ar();
        int b2 = Bar$.MODULE$.b/*caret*/ar();
        Bar$.MODULE$.bar_/*caret*/$eq(3);
        Bar2.bar_$/*caret*/eq(2);
        Bar2.ba/*caret*/r();
        Bar2$.MODULE$.b/*caret*/ar();
        Bar2$.MODULE$.bar_/*caret*/$eq(4);
    }
}
